apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/priv/PrivateChatsView.java

```diff
@@ -292,6 +287,28 @@ private VBox createUserProfileVBox(UserProfileDisplay userProfileDisplay, String
         return vbox;
     }
 
+    private void updateAndShowMyProfile(UserProfile userProfile, PrivateChatsModel model) {
+        if (null == chatMyUserProfileDisplay) {
+            chatMyUserProfileDisplay = new UserProfileDisplay(25);
+        }
+        chatMyUserProfileDisplay.setReputationScore(model.getMyUserReputationScore());
+        chatMyUserProfileDisplay.setUserProfile(userProfile);
+        chatMyUserProfileDisplay.setReputationScore(model.getMyUserReputationScore());
+        chatMessagesComponent.getChildren().stream()
+                .filter(node -> "chat-messages-bottom-bar-container".equals(node.getId()))
```

Comment:
We do not access UI components by ID but use their reference instead. I have not looked into that particular case but I am pretty sure there is a more robust way to do it.

Reply:
Reallt appriciate your review and comment though this PR has been closed.